scripts/templates/typescript-project/config/default.yaml

```diff
@@ -0,0 +1,109 @@
+#
+# %APP_NAME% - default.yaml
+#
+# This file includes all the configuration that every environment has in common. Every environment
+# (named by the NODE_ENV environment variable) can have its own configuration file. If the
+# environment specific configuration file overlaps entries with this file, the environment specific
+# one will override the values here.
+#
+
+# Define all apps in existence. Generally our game and dashboard apps (but we leave those to the
+# individual environments).
+
+apps:
+    # Our main app: the game
+    game:
+        # Number of seconds to keep user command responses cached (to mitigate weak client connections)
+        responseCache: 30
+
+        # The maximum access level for this app (user commands with a higher level are not exposed)
+        access: user
```

Comment:
I did. This is the typescript template, not the javascript one, and this was started prior to your other PR being made. Will fix it.